torq.q

```diff
@@ -185,7 +185,7 @@ getapplication:{$[0 = count a:@[{read0 x};hsym last getconfigfile"application.tx
 // Logging functions live in here
 
 // Format a log message
-format:{[loglevel;proctype;proc;id;message] "|" sv (string .proc.cp[];string .z.h;string proctype;string proc;string loglevel;string id;message)}
+format:{[loglevel;proctype;proc;id;message] "|" sv (string .proc.cp[];string .z.h;string proctype;string proc;string loglevel;string id;$[1=count message;enlist message; message])}
```

Comment:
This wont fix it - what if they pass in a 1 item list? 
If you want to force it to be a list, better to join with the empty list

Reply:
Can this be cleaned up while we're here?
```suggestion
format:{[loglevel;proctype;proc;id;message] "|"sv string[(.proc.cp[];.z.h;proctype;proc;loglevel;id)],enlist(),message}
```

Reply:
Can this be cleaned up while we're here?
```suggestion
format:{[loglevel;proctype;proc;id;message]"|"sv string[(.proc.cp[];.z.h;proctype;proc;loglevel;id)],enlist(),message}
```